model-optimizer/mo/utils/get_ov_update_message.py

```diff
@@ -21,9 +21,9 @@
 
 
 def get_ov_update_message():
-    expected_update_date = datetime.date(year=2021, month=4, day=1)
+    expected_update_date = datetime.date(year=2021, month=7, day=1)
     current_date = datetime.date.today()
 
-    link = 'https://software.intel.com/content/www/us/en/develop/tools/openvino-toolkit/choose-download.html?cid=other&source=Prod&campid=ww_2021_bu_IOTG&content=upg_pro&medium=organic_uid_agjj'
+    link = 'https://software.intel.com/content/www/us/en/develop/tools/openvino-toolkit/download.html?cid=other&souce=prod&campid=ww_2021_bu_IOTG_OpenVINO-2021-3&content=upg_all&medium=organic'
```

Comment:
Is it a typo in the word "source"? `souce=prod`

Reply:
Thanks, @lazarevevgeny